pipeline/alert.go

```diff
@@ -1254,3 +1259,37 @@ func (a *AlertNode) Talk() *TalkHandler {
 type TalkHandler struct {
 	*AlertNode
 }
+
+// Send the alert to Kafka.
+// To use Kafka alerting you must have a kafka instance running.
+// Add the url, port and the brokers that alerts should go to.
+// You can also optionally enable SASL or TLS authentication.
+//
+// Example:
+//    [kafka]
+//      enabled = true
+//      brokers = "kafka.example.com:9092,kafka2.example.com:8080"
+//
+// Example:
+//    stream
+//         |alert()
+//             .kafka()
+//
+// Send alerts to Kafka.
+//
+// tick:property
+func (a *AlertNode) Kafka(topics []string) *KafkaHandler {
```

Comment:
This function signature should be:

`func (a *AlertNode) Kafka(topics ...string) *KafkaHandler {`

The `...` means is a [variadic](https://en.wikipedia.org/wiki/Variadic_function) function.